duckduckgo/macos-browser
# Fix link type check

Task/Issue URL: https://app.asana.com/0/0/1203270151638332/f
Tech Design URL:
CC: @kdzwinel 

**Description**:
This PR fixes the activation of link protection to operate on navigations caused by `window.location = ...`



**Steps to test this PR**:
1. Navigate to https://privacy-test-pages.glitch.me/privacy-protections/
1. Open the dev console and enter `window.location = 'https://privacy-test-pages.glitch.me/privacy-protections/query-parameters/query.html?utm_source=something&q=other'`
2. Result should match the expected string from the first page

**Testing checklist**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
